experiments/compositions/composition/expanders/jinja2/main.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	pb "google.com/composition/proto"
+	"google.golang.org/grpc"
+	"tailscale.com/atomicfile"
+)
+
+var (
+	port = flag.Int("port", 50051, "The server port")
```

Comment:
This implies each instance (eg version 1.1 and 1.2) run in its own pod.

Reply:
Yes it does. For now keeping it simple. We can in the future look at other options. This allows us to run different images for different pods instead of a shared image (for CVE mitigation).